src/components/inputs/Selector.tsx

```diff
@@ -214,7 +215,8 @@ const AssigneeSelectorRenderer = ({ props, option }: { props: HTMLAttributes<HTM
       }}
     >
       <Stack direction="row" alignItems="center" columnGap={3}>
-        <Avatar
+        <AvatarWithInitials
+          altName={(option as IAssigneeCombined)?.givenName || (option as IAssigneeCombined)?.familyName}
           alt="user"
           src={(option as IAssigneeCombined).avatarImageUrl || (option as IAssigneeCombined).iconImageUrl}
```

Comment:
```suggestion
const assignee = option as IAssigneeCombined

return (
        ...
        <AvatarWithInitials
          altName={assignee?.givenName || assignee?.familyName}
          alt="user"
          src={assignee.avatarImageUrl || assignee.iconImageUrl}
          ...
)
```

Also, I think we have a schema for combined assignee used in the backend `types` folder somewhere as well, which could be used instead of the `option as IAssigneeCombined` type casting, but this should be fine for now too.